### PR TITLE
PPF-53 Added activation requested and integration deleted mail

### PR DESCRIPTION
**Pull request by grubolsch**
### Added
- Added "integration deleted" email.
- Added "Integration activation requested email"

### Fixed
- Typo in method name "withReminderEmailSent"

---
Ticket: https://jira.uitdatabank.be/browse/PPF-53

### CODE REVIEW

File: app/Domain/Mail/MailManager.php [hunk 1]
```diff
@@ -51,6 +53,20 @@ public function sendIntegrationBlockedMail(IntegrationBlocked $event): void
         $this->sendMail($integration, $this->templates->getOrFail(Templates::INTEGRATION_BLOCKED));
     }
 
+    public function sendIntegrationActivationRequestMail(IntegrationActivationRequested $event): void
```
Comment (LucWollants) [inline app/Domain/Mail/MailManager.php:56]: Is it not weird to send an email to an integrator when he/she requests the activation? I only see a request from Corneel to send a mail for the delete action, see https://jira.publiq.be/browse/PPF-53?focusedId=137354&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-137354

Comment (grubolsch) [reply app/Domain/Mail/MailManager.php:57]: This is not noted in a ticket yet, it comes from the workdocument for the email templates: https://docs.google.com/document/d/1ly2Mg-pe3WjcrFLO61AybeL8kOPonulTBcDQPaBPNyY/edit?usp=sharing
I gave you access normally right now.

Comment (LucWollants) [reply app/Domain/Mail/MailManager.php:57]: Ok, there is indeed a section about this type of mail. I guess it's best to update the Jira ticket to also have a section about the activation request.

Comment (LucWollants) [reply app/Domain/Mail/MailManager.php:57]: According to this document, the blocked mail can be removed.

**Review by LucWollants — APPROVED** [review]
Nice work! It is easy to add new template flavors and alot of the boiler plate code is typesafe.

**Pull request merged**